test-resources/sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.projectKey=mobile-id-check-async-test-resources
+sonar.organization=govuk-one-login
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+sonar.sources=src
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
+sonar.language=ts
+sonar.exclusions=**/*.test.ts,**/*.test.js,**/testUtils/**,**/tests/mocks.ts
```

Comment:
Please may we tailor this for this application

Reply:
Updated